src/TransactionController.ts

```diff
@@ -428,6 +430,37 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 		this.update({ transactions });
 	}
 
+	async stopTransaction(transactionID: string) {
+		const transactionMeta = this.state.transactions.find(({ id }) => id === transactionID);
+		if (!transactionMeta) {
+			return;
+		}
+
+		if (!this.sign) {
+			throw new Error('No sign method defined.');
+		}
+
+		const existingGasPrice = transactionMeta.transaction.gasPrice;
+		/* istanbul ignore next */
+		const existingGasPriceDecimal = parseInt(existingGasPrice === undefined ? '0x0' : existingGasPrice, 16);
+		const gasPrice = `0x${(existingGasPriceDecimal * 1.5).toString(16)}`;
```

Comment:
We multiply the previous gas price by `1.5`, but this is pretty arbitrary. I'm checking into what the extension does so we stay uniform...